defaults/main.yml

```diff
@@ -13,5 +13,17 @@ letsencrypt_certbot_verbose: true
 letsencrypt_pause_services: []
 # Force Certificate Reneval
 letsencrypt_force_renew: true
-# Additional certbot arguments
-letsencrypt_certbot_args: []
+# certbot required parameters
+letsencrypt_certbot_default_args:
+  - "certonly"
```

Comment:
If you keep this YAML list style, you should be able to drop the `"`'s afaict, though I guess it doesn't matter that much.